ocamldoc/odoc_html.ml

```diff
@@ -2618,15 +2618,17 @@ class html =
       try
         let chanout = open_out (Filename.concat !Global.target_dir self#index) in
         let b = new_buf () in
-        let title = match !Global.title with None -> "" | Some t -> self#escape t in
         bs b doctype ;
         bs b "<html>\n";
         self#print_header b self#title;
         bs b "<body>\n";
 
-        bs b "<h1>";
-        bs b title;
-        bs b "</h1>\n" ;
+        let () = match !Global.title with
+          | None -> ()
+          | Some t ->
+              let () = bs b "<h1>" in
+              let () = bs b (self#escape t) in
+              bs b "</h1>\n" in
```

Comment:
Sorry for the nitpicking but the `let () = …  in` style is quite at odd with the rest of the file, even more so when printing the title. I am little bit worried that potential future readers would be left wondering why the `;` operator was so violently avoided in this specific part of the code.